proofzero/rollupid
# fix(console): Deleting identity groups with apps fails

### Description

- Added UI-level preventive modal when trying to delete group that has apps associated
- Added platform-level prevention mechanism
- Fixed display bug for application list

### Related Issues

- Closes #2679 

### Testing

1. Create group
2. Create app as part of group
3. Go to groups list
4. Open context menu
5. Press delete
6. See modal

<img width="372" alt="image" src="https://github.com/proofzero/rollupid/assets/635806/a0d27f33-1384-46bd-a0f4-02a5141455ec">

### Checklist
